doc/source/whats_new.rst

```diff
@@ -106,6 +106,8 @@ Changelog
 
    - Add support for custom sensor positions when creating Layout objects by `Chris Holdgraf`_
 
+   - Add concatenated epoch plotting where user can select bad epochs interactively by `Jaakko Leppakangas`_
```

Comment:
How about this: "Add epochs browser to interactively view and manipulate epochs" ?